actionview/lib/action_view/path_registry.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module ActionView # :nodoc:
```

Comment:
I don't think we need this nodoc (the one on `PathRegistry` we do though)

Reply:
😅 the one file that I based this on was `PathSet` and _that_ had a `nodoc`. I guess I should also remove that for consistency

Reply:
Actually there are more modules set with `nodoc` 🤔 should I also remove those?

<img width="743" alt="Screen Shot 2023-02-23 at 4 57 02 PM" src="https://user-images.githubusercontent.com/1518902/221065777-49933fd3-76d9-46ff-a1c6-49cc52f07501.png">

Reply:
NVM then I'm wrong 😅